### PR TITLE
Support camera.imageRegion parameter

### PR DESCRIPTION
**Pull request by szellmann**

### CODE REVIEW

File: Camera.cpp [hunk 1]
```diff
@@ -11,5 +11,17 @@ Camera::Camera(std::string subtype)
...
 {}
 
+void Camera::setParam(const char *_id, ANARIDataType type, const void *mem)
+{
+  std::string id(_id);
+
+  if (id == "imageRegion") {
+    Object::setParam("imageStart", ANARI_FLOAT32_VEC2, (const float *)mem);
+    Object::setParam("imageEnd", ANARI_FLOAT32_VEC2, ((const float *)mem)+2);
+  }
```
Comment (johguenther) [inline Camera.cpp:21]: With an `else` we avoid the "unknown param 'imageRegion'" error from OSPRay.

Comment (szellmann) [reply Camera.cpp:21]: Yes indeed. I realized that, too, but forgot to add to the patch. Could you just add the `else` there or should I?

Comment (johguenther) [reply Camera.cpp:21]: I'll do it. In general we will merge to the internal repo first and then push back to GitHub.

**Comment by johguenther** [comment]
merged manually

**Pull request closed without merging**